scripts/data_augment/data_augment.py

```diff
@@ -0,0 +1,464 @@
+import torch
```

Comment:
```suggestion
'''What does this script do

example usage: 

args: 
'''

import torch
```

maybe add some small description and example usage and maybe args at top of file so easy for others to just get quick idea of what its doing.

Reply:
Added the comment, thanks for your feedback